### PR TITLE
Feature/looker studio template public

### PR DESCRIPTION
**Pull request by anuradha-bajpai-google**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -595,8 +595,8 @@ the specified threshold limit.
     To upgrade existing installations:
 
     *   Re-run the Terraform, to update the Cloud Functions and Scheduled Query
-    *   Update the SQL used in the Data Studio dashboard according to Step #7
-        of [3.10 Data Studio Dashboard setup](#310-data-studio-dashboard-setup).
+    *   Update the SQL used in the Looker Studio dashboard according to Step #7
+        of [3.10 Looker Studio Dashboard setup](#310-data-studio-dashboard-setup).
```
Comment (bgood) [inline README.md:599]: `#310-data-studio-dashboard-setup` needs to be updated to `#310-looker-studio-dashboard-setup`

**Review by bgood — CHANGES_REQUESTED** [review]
Need to fix one link in the readme.

File: README.md (hunk 1 above)
Comment (anuradha-bajpai-google) [reply README.md:599]: Already fixed in the second commit

Comment (bgood) [reply README.md:599]: The second commit changes it on line 82, but it also needs to be updated on line 599.

Comment (anuradha-bajpai-google) [reply README.md]: Thanks Ben! Updated the link and all checks passed.

**Review by bgood — APPROVED** [review]

**Pull request merged**